crates/astria-sequencer-relayer/src/relayer/celestia_client/builder.rs

```diff
@@ -40,6 +43,9 @@ pub(in crate::relayer) enum BuilderError {
     /// The node info response was empty.
     #[error("the celestia node info response was empty")]
     EmptyNodeInfo,
+    /// Mismatch in Celestia chain ID.
+    #[error("mismatch in celestia chain id, configured id: `{configured}`, actual id: `{actual}`")]
```

Comment:
I am frequently confusing myself when seeing `actual`. Maybe `received` to make it clear that this was read off the remote host?
```suggestion
    #[error("mismatch in celestia chain id, configured id: `{configured}`, received id: `{actual}`")]
```

Reply:
Done in [458f2a2](https://github.com/astriaorg/astria/pull/1063/commits/458f2a27ff3a3fe97880a3b049c4359c1078058f).